components/directions/DirectionList.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from 'next/link';
+
+import { COLOR_THEME } from '@/constants/color';
+
+import { Direction } from '@/types/directions';
+import { directions } from '@/types/page';
+
+import { getPath } from '@/utils/page';
+
+import CornerFoldedRectangle from '../common/CornerFoldedRectangle';
+
+interface DirectionsListProps {
+  directionsList: Direction[];
+  selectedDirections?: Direction;
```

Comment:
단일 Direction 같은데 selectedDirection은 어때

```suggestion
  selectedDirection?: Direction;
```